content/FIDO2/FIDO2_WebAuthn_Developer_Guide/Desktop_and_Mobile_Authentication.adoc

```diff
@@ -0,0 +1,14 @@
+== Desktop and Mobile Authentication
+=== Desktop App/Platform Authentication
+Yubico’s https://github.com/Yubico/libfido2[libfido2] and https://github.com/Yubico/python-fido2[python-fido2] open source libraries provide examples for how to communicate with a FIDO device over USB, and can verify attestation and assertion signatures. Both FIDO U2F (CTAP1) and FIDO2 (CTAP2) protocols are supported on Linux, MacOS, and Windows platforms
+
+https://blogs.windows.com/msedgedev/2018/07/30/introducing-web-authentication-microsoft-edge/#UZcrEz8uWr1dcz0R.97[Microsoft Edge announced Web Authentication support] which enables FIDO2 devices or Windows Hello users to authenticate without any password on any Windows 10 device, using biometrics or a PIN. The https://docs.microsoft.com/en-us/microsoft-edge/dev-guide/windows-integration/web-authentication[Web Authentication and Windows Hello dev guide] will walk through the flow. This guide highlights how you can detect if Windows Hello is available on a PC.  And how you can set the authenticatorAttachment parameter for the best user experience when registering a FIDO2 capable platform, such as Windows 10.
+
+=== Mobile App/Platform Authentication
+==== Android
+Yubico has an https://github.com/Yubico/android-u2f-demo[open source android U2F demo application], which uses the platform’s U2F libraries to demonstrate FIDO U2F over NFC. Android’s FIDO2 platform libraries are still under development.
```

Comment:
This is incorrect. This U2F demo app does not use the platform APIs (it predates them by quite a bit) and should not be referenced at all at this point. The official Android APIs for FIDO2 (https://developers.google.com/android/reference/com/google/android/gms/fido/fido2/Fido2ApiClient) aren't very well documented, but are functional (though currently limited to CTAP1).

Reply:
Thanks, removed from eacd866